save_results

```diff
@@ -179,15 +191,16 @@ else
 	echo commit: ${version} > $RESULTS_PATH/version
 fi
 pushd $export_results > /dev/null
-cp ${curdir}/meta_data*.yml ${RESULTS_PATH}
-cp ${curdir}/hw_info.out ${RESULTS_PATH}
+
+copy_file ${curdir}/meta_data*.yml ${RESULTS_PATH}
+copy_file ${curdir}/hw_info.out ${RESULTS_PATH}
 
 tar hcf results_${test_name}_${tuned_setting}.tar ${results_dir}
-cp -R ${results_dir}/* .
-ln -s results_${test_name}_${tuned_setting}.tar results_pbench_${test_name}_${tuned_setting}.tar
+copy_file -R ${results_dir}/* .
```

Comment:
Wouldn't `-R` become `$1` in `copy_file`?  Something tells me that many system would not have `-R` as a file

Reply:
Not sure where my comments went.
Fixed, We do a cp -R in copy_file now.  If copying a file, the -R is a no op.  We also have to check $2 for being null.